Libraries/Components/TextInput/TextInput.js

```diff
@@ -378,6 +382,15 @@ const TextInput = React.createClass({
      */
     selectionState: PropTypes.instanceOf(DocumentSelectionState),
     /**
+     * The start and end of the text input's selection. Set start and end to
+     * the same value to position the cursor.
+     * @platform ios
+     */
+    selection: PropTypes.shape({
+      start: PropTypes.number.isRequired,
+      end: PropTypes.number.isRequired
```

Comment:
Sure, I'll just add this logic in JS `render` so native can still assume both values are set.